pkg/gtk4-layer-shell/build.zig.zon

```diff
@@ -0,0 +1,15 @@
+.{
+    .name = "gtk4-layer-shell",
+    .version = "1.1.0",
+    .paths = .{""},
+    .dependencies = .{
+        .gtk4_layer_shell = .{
+            .url = "https://github.com/wmww/gtk4-layer-shell/archive/refs/tags/v1.1.0.tar.gz",
+            .hash = "12203eff4829ad8afdd828eb323d48e5ba8dbb44d224e9e314d4ab1533c2bec20f4b",
+        },
+        .wayland_protocols = .{
+            .url = "https://deps.files.ghostty.org/wayland-protocols-258d8f88f2c8c25a830c6316f87d23ce1a0f12d9.tar.gz",
+            .hash = "12201a57c6ce0001aa034fa80fba3e1cd2253c560a45748f4f4dd21ff23b491cddef",
+        },
```

Comment:
Since there's probably not an automatic way to do this, we should add some comments here reminding people to sync this dep with the same dep in the main build.

Reply:
Hmm... The cursed thing is that the repo itself vendors *some* protocols (i.e. the wlr-layer-shell protocol itself) but for the core/XDG protocols it relies on the wayland-protocols repo, probably for the higher stability guarantees. I think the main benefit of syncing would be dependency deduplication, but I don't know if Zig currently does that...

Reply:
If there's an explicit reason to keep them out of sync that should be documented in a comment. A zealous maintainer might update them without realizing. Otherwise I feel that they should be kept in sync.

Reply:
Yeah, I'm also more inclined to just update it whenever it breaks (e.g. when upstream relies on protocols that are newer than what are in our copy of wayland-protocols)